veritable-react-gui/src/components/Common/Navigation/Connectivity/ConnectivityWrap/ConnectivityWrap.js

```diff
@@ -34,14 +35,53 @@ export default function ConnectivityWrap({
 
   const [statusDelete, deleteError, startDelete] = useDeleteConnections()
 
+  const isConnected = (agent, data = false) => {
+    if (!data) return false
+    const connectionFound = data.find(({ their_label, state }) => {
+      const isCAA = their_label.includes(agent)
+      const isActive = state === 'active'
+      return isCAA && isActive
+    })
+
+    if (!connectionFound) return false
+
+    setCAAConnected(true)
+    return true
+  }
+
+  // only for holder
+  // other personas should not have or need this function
+  const createConnectionToCAA = (label = 'authority', data) => {
+    const connected = isConnected(label, data?.results)
+    if (personaPrefix !== 'licensee' || connected) {
+      return null
+    }
+    // TODO origin should be an env var for CAA
```

Comment:
Todo?

Reply:
mm... this was a test:) you have passed, going to check. To be fair I left it because there was no config or .env implementation so did not want to do something that might be outside of the scope. Happy to add `config` or either `.env`. What are your thoughts? It could also become a `constant`

Reply:
Whilst I'm not convinced by the pattern elsewhere it's probably good to stick with it. If you look at other components @n3op2 you'll see `origin` is passed in as a prop and fed down. That is what we should probably do here